core/server/utils/url.js

```diff
@@ -229,7 +229,7 @@ function urlFor(context, data, absolute) {
             secure = data.author.secure;
         } else if (context === 'image' && data.image) {
             urlPath = data.image;
-            imagePathRe = new RegExp('^' + getSubdir() + '/' + config.get('paths').imagesRelPath);
+            imagePathRe = new RegExp('^' + getSubdir() + '/' + module.exports.staticImageUrlPrefix);
```

Comment:
this is weird!
we could add a var on top and use it for export and internal file usage?